config/protocol.privnet.docker.four.yml

```diff
@@ -78,6 +78,8 @@ ApplicationConfiguration:
   Pprof:
     Enabled: false
     Port: 20014
-  UnlockWallet:
-    Path: "/wallet4.json"
-    Password: "four"
+  Consensus:
+    Enabled: true
+    UnlockWallet:
+      Path: "/wallet4.json"
+      Password: "four"
```

Comment:
We don't have this section in mainnet\neofs\testnet config files, maybe it's time to add it? With `Enabled: false`, just for user information purposes.

Reply:
Yeah, I've had this thought as well, it makes some sense, yet at the same time it's not something used often for mainnet/testnet.